Testscripts/Windows/GPU-DRIVER-INSTALL.ps1

```diff
@@ -245,7 +258,14 @@ function Main {
         # The expected ratio is 1 GPU adapter for every 6 CPU cores
         $vmCPUCount = Run-LinuxCmd -username $user -password $password -ip $allVMData.PublicIP `
             -port $allVMData.SSHPort -command "nproc" -ignoreLinuxExitCode
-        [int]$expectedGPUCount = $($vmCPUCount/6)
+
+        if ($allVMData.InstanceSize -match "Standard_NDv2") {
```

Comment:
do we have any official doc for those matches?

Reply:
asking...

Reply:
comments are enough

Reply:
done in 2db0bf2b079fc0bd7dca9558d62a40468f897fe4